NXP-22287: fix drive edit url

adilioaraujo opened the pull request.

nuxeojenkins commented:
From the blame information on this pull request, potential reviewers: @nelsonsilva, @guirenard

Gabez0r commented:
👍

adilioaraujo commented:
- https://github.com/nuxeo/nuxeo-elements/pull/13/files was merged. 
- added `user` to BLOB_ACTIONS slot (https://github.com/nuxeo/nuxeo-web-ui/commit/1ee37c2d1a29eb5bdf14c65c3bcbb38b352a4108).
- removed `nuxeo-connection` and replaced `this.$.nx.user.id` with `user.id` (https://github.com/nuxeo/nuxeo-drive-server/pull/30/commits/1c8b60d8e1907c2c11684b274e36c0feff1326d9).
